src/Symfony/Component/Lock/Tests/Store/FlockStoreTest.php

```diff
@@ -56,11 +56,8 @@ public function testConstructWhenRepositoryIsNotWriteable()
 
     public function testConstructWithSubdir()
     {
-        if (!getenv('USER') || 'root' === getenv('USER')) {
-            $this->markTestSkipped('This test will fail if run under superuser');
-        }
-
-        new FlockStore(sys_get_temp_dir().'/sf-flock');
+        new FlockStore($dir = (sys_get_temp_dir().'/sf-flock'));
+        $this->assertDirectoryExists($dir);
```

Comment:
What about removing the directory afterwards?

Reply:
done